src/Features/CSharp/Portable/CSharpFeaturesResources.Designer.cs

```diff
@@ -296,6 +296,15 @@ internal class CSharpFeaturesResources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Convert &apos;foreach&apos; to &apos;for&apos;.
+        /// </summary>
+        internal static string Convert_foreach_to_for {
+            get {
+                return ResourceManager.GetString("Convert_foreach_to_for", resourceCulture);
+            }
+        }
```

Comment:
what about .resx file?

Reply:
not sure... looks like something messed up while I do merge conflict.